screens/Home/IntroSlidersScreen.tsx

```diff
@@ -80,14 +82,19 @@ export const IntroSlidersScreen: React.FC<RootRouteProps> = (props) => {
           </Row>
           <Image source={item.image} />
           <Column
+            testID="introSlideColumn"
```

Comment:
can we name it better?
"introSlide" will be better instead of "introSlideColumn"

Reply:
done